frontend/src/metabase/ui/components/inputs/Switch/Switch.styled.tsx

```diff
@@ -116,5 +116,13 @@ export const getSwitchOverrides = (): MantineThemeOverride["components"] => ({
         },
       };
     },
+    variants: {
```

Comment:
Could you add a block in the component story for this variant? And a control to toggle variants? And please ask @kdoh to review